components/gantt/gantt-tree/columns/resize.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Resize
+page_title: TreeList - Resize Columns
+description: Drag to resize columns in the treelist for Blazor.
```

Comment:
wrong component name in title and description

Reply:
I do not need to see this again for second approval

Reply:
Done, thank you for the feedback